### ISSUE TITLE
A11y: All page content should be contained by Landmarks. 

### ISSUE THREAD

**Issue by kimclow** [Automattic/wp-calypso#79281]
### What

Ensures all page content is contained by landmarks. 

### How to reproduce the issue

Install This [Extention](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd)
Sign in with demo/temp mail generator to get premium facilities.
Go to a WordPress site URL eg: https://opencoresummit.com
Open the dev tool and launch Axe core
Start the Scan page and check the overview tab Error Segment.

### Why

Navigating a web page is far simpler for screen reader users if the content splits between multiple high-level sections. Content outside of sections is difficult to find, and the content's purpose may be unclear.

### How

There is a lot of helpful information and advice on this web page: 
https://dequeuniversity.com/rules/axe/4.7/region?application=AxeChrome

**Comment by kimclowa8c** [comment]
Here is a screenshot from https://brian-ho-production.mystagingwebsite.com/  listed above: 

![245923483-df78e526-8305-4407-be54-1c2a1cd162d2](https://github.com/Automattic/wp-calypso/assets/83212200/1703505c-5e49-4251-a309-f2086e4a4b62)

**Comment by kimclow** [comment]
And here are screenshots for the Open Core Summit referenced above: 

<img width="1439" alt="238691127-31fd808a-4e34-4a1e-a288-d3f16a563abd" src="https://github.com/Automattic/wp-calypso/assets/83212200/fd42e607-7a64-4016-89b9-b541d13bb871">

![238691150-9bcde037-62dd-4522-a91c-2cbdaaad6cf4](https://github.com/Automattic/wp-calypso/assets/83212200/5973b67f-0feb-480f-bfff-6caf21bbcf65)

**Comment by kimclow** [comment]
I think I've reported this in the incorrect repo. Apologies for the noise.
